content/geometry/geometry-index.adoc

```diff
@@ -0,0 +1,65 @@
+= Vehicle structure
+:toc:
+:toclevels: 4
+
+
+include::vehicle-structure/introduction.adoc[leveloffset=+1]
```

Comment:
In my opinion it is a bit counter-intuitive to add individual elements of a certain object structure to the general geometry index. I would have expected to find this index in the vehicle-structure folder for the elements of the vehicle structure. Is that possible?